src/components/Navigation/Navigation.test.js

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react'
+import Navigation from './Navigation'
+import user from '@testing-library/user-event'
+
+describe('Navigation', () => {
+  it('renders correctly', () => {
+    render(<Navigation onClick={() => {}}>all songs</Navigation>)
+  })
```

Comment:
Did you intend to run a snapshot test? It is not testing at the moment, isn't? Just rendering?

Reply:
Thanks for the hint! :D